tests/integration/helm/install_test.go

```diff
@@ -40,18 +42,54 @@ global:
 	framework.
 		NewTest(t).
 		Features("installation.helm.default.install").
-		Run(setupInstallation(overrideValuesStr, false))
+		Run(setupInstallation(overrideValuesStr, false, DefaultNamespaceConfig))
 }
 
 // TestAmbientInstall tests Istio ambient profile installation using Helm
 func TestAmbientInstall(t *testing.T) {
 	framework.
 		NewTest(t).
 		Features("installation.helm.ambient.install").
-		Run(setupInstallation(ambientProfileOverride, true))
+		Run(setupInstallation(ambientProfileOverride, true, DefaultNamespaceConfig))
+}
+
+func TestAmbientInstallMultiNamespace(t *testing.T) {
+	tests := []struct {
+		name     string
+		nsConfig NamespaceConfig
+	}{{
+		name: "istio-cni",
```

Comment:
```suggestion
		name: "isolated-istio-cni",
```

IIUC might be good to use a more descriptive case name here.